WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -2320,10 +2321,15 @@ private void updateCurrentTagIfTime() {
         new Thread() {
             @Override
             public void run() {
+                // Check the fragment is attached to the activity when this Thread starts.
+                FragmentActivity activity = getActivity();
+                if (activity == null) {
```

Comment:
💡 I think doing the check here still has a great chance of not solving the crash, since the function that probably takes the most time to run is `shouldAutoUpdateTag`. This means that even with this check here, when the code reaches line 2330 or 2332 the fragment might already be dettached anyway, which will likely yield other issues.

Because of that, I think it would be better if the check is done in each of the if clause's blocks (before line 2330 and line 2332) below.

I know the code will look a bit repetitive, but since we're working with Java here, there's not much way around it. 😞